packages/kbn-types/example/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
```

Comment:
__optional nit:__ we should probably update `package-lock.json` for `kbn-types` and `kbn-types/example`.